velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -703,12 +765,20 @@ TypePtr VectorFuzzer::randType(int maxDepth) {
       VARBINARY(),
       TIMESTAMP(),
       DATE(),
-  };
-  static constexpr int kNumScalarTypes =
-      sizeof(kScalarTypes) / sizeof(kScalarTypes[0]);
+      SHORT_DECIMAL(ShortDecimalType::kMaxPrecision, 0),
```

Comment:
Can we generate decimals with different precisions and scales?

Reply:
This is just a place holder. The precision and scale will be randomly generated here: https://github.com/facebookincubator/velox/pull/3768/files/34d539eca6b0616c69ce00d0a2e5c6aaf93ac265#diff-68474a90a74f69bf4c82ba43c95a744abcf7e95c1ff6ffd31cae8cda84b5250bR778

Explained in comments.